sdks/python/apache_beam/examples/wordcount_debugging.py

```diff
@@ -48,25 +48,22 @@
 import apache_beam as beam
 from apache_beam.io import ReadFromText
 from apache_beam.io import WriteToText
+from apache_beam.metrics import Metrics
 from apache_beam.utils.pipeline_options import PipelineOptions
 from apache_beam.utils.pipeline_options import SetupOptions
 
 
 class FilterTextFn(beam.DoFn):
   """A DoFn that filters for a specific key based on a regular expression."""
-
-  # A custom aggregator can track values in your pipeline as it runs. Those
-  # values will be displayed in the Dataflow Monitoring UI when this pipeline is
-  # run using the Dataflow service. These aggregators below track the number of
-  # matched and unmatched words. Learn more at
-  # https://cloud.google.com/dataflow/pipelines/dataflow-monitoring-intf about
-  # the Dataflow Monitoring UI.
-  matched_words = beam.Aggregator('matched_words')
-  umatched_words = beam.Aggregator('umatched_words')
-
   def __init__(self, pattern):
     super(FilterTextFn, self).__init__()
     self.pattern = pattern
+    # A custom metric can track values in your pipeline as it runs. Those
```

Comment:
drop Dataflow. 

maybe instead of `Those values will be displayed in the Dataflow Monitoring UI when this pipeline is run using the Dataflow service.`

`Those values will be available in the monitoring system of the runner used to run this pipeline.`